library/src/test/java/com/vinted/coper/CoperImplTest.kt

```diff
@@ -30,10 +30,9 @@ import kotlin.test.assertTrue
 @Config(manifest = Config.NONE, sdk = [23, 27])
 class CoperImplTest {
 
-    private val activityController = Robolectric.buildActivity(FragmentActivity::class.java)
-    private val activity: FragmentActivity = spy(activityController.setup().get())
-    private val shadowActivity = shadowOf(activity)
-    private val fixture: CoperImpl = spy(getCoperInstance())
+    private val activityController = Robolectric.buildActivity(CoperTestActivity::class.java)
+    private val activity = activityController.setup().get()
+    private val fixture: CoperImpl = getCoperInstance()
```

Comment:
Was `CoperImpl` field declaration is left intentionally? I Think that compiler can figure the type out.

Reply:
For this case I left it intentionally, because getCoperInstance is a method, which could return anything. What do you think?